src/goLiveErrors.ts

```diff
@@ -0,0 +1,88 @@
+'use strict';
+
+import vscode = require('vscode');
+import { byteOffsetAt, getBinPath } from './util';
+import cp = require('child_process');
+import path = require('path');
+import { promptForMissingTool } from './goInstallTools';
+import { errorDiagnosticCollection } from './goMain';
+
+// Interface for settings configuration for adding and removing tags
+interface GoLiveErrorsConfig {
+	delay: number;
+	enabled: boolean;
+}
+
+let runner;
+
+export function goLiveErrorsEnabled() {
+	let goConfig = <GoLiveErrorsConfig>vscode.workspace.getConfiguration('go')['liveErrors'];
+	if (goConfig === null || goConfig === undefined || !goConfig.enabled) {
+		return false;
+	}
+	let autoSave = vscode.workspace.getConfiguration('files')['autoSave'];
+	if (autoSave !== null && autoSave !== undefined && autoSave !== 'off') {
```

Comment:
I was erring on the side of caution.

Good point on the fact that auto save onFocusChange and onWindowChange has no affect whatsoever.

I was concerned with the autosave on delay. Then you have build and gotype both running and giving essentially similar results and overriding each other.

On second thoughts, it is definitely being over cautious. We can remove the check in the next update.